src/shipit_taskcluster/requirements.txt

```diff
@@ -0,0 +1,2 @@
+-e ./../../lib/cli_common[all]#egg=mozilla-cli-common
+-e ./../../lib/backend_common[api,cors,auth,db]#egg=mozilla-backend-common
```

Comment:
pip also complains
 `   mozilla-cli-common 1.0.0 does not provide the extra 'all'`
Perhaps just `[taskcluster]`, although we also get it from backend_common via auth.